cinema.java

```diff
@@ -82,14 +175,20 @@ static int seatNum() {
         return seatNum;
     }
 
-
+    /**
+     * Prints the program menu.
+     */
     static void printMenu() {
         System.out.println("1. Show the seats");
         System.out.println("2. Buy a ticket");
+        System.out.println("3. Statistics");
         System.out.println("0. Exit");
     }
 
-    static void menu(int numOfSeats, int numOfRows, int totalSeats, char[][] seatLayout) {
+    /**
+     * Program Logic for the menu.
+     */
+    static void menu(int numOfSeats, int numOfRows, int totalSeats, char[][] seatLayout, int soldTickets) {
```

Comment:
Would it be reasonable to have `case 0:` instead? Not sure what that would have to change but it's a thought